core/dbt/events/types.py

```diff
@@ -264,3 +306,4 @@ def cli_msg(self) -> str:
     SystemStdOutMsg(bmsg=b'')
     SystemStdErrMsg(bmsg=b'')
     SystemReportReturnCode(code=0)
+    SelectorReportInvalidSelector(selector_methods={'':''}, spec_method='', raw_spec='')
```

Comment:
missing `SelectorAlertAllUnusedNodes` and `SelectorAlertUpto3UnusedNodes`

Reply:
good catch!